autorest/codegen/templates/lro_operation.py.jinja2

```diff
@@ -57,13 +58,15 @@ response_headers = {
         'polling_interval',
         self._config.polling_interval
     )
-    raw_result = {{ keywords.await }}self._{{ operation.name }}_initial(
-        {% for parameter in operation.parameters.method %}
-        {{ parameter.serialized_name }}={{ parameter.serialized_name }},
-        {% endfor %}
-        cls=lambda x,y,z: x,
-        **kwargs
-    )
+    cont_token = kwargs.pop('continuation_token', None)  # type: Optional[str]
```

Comment:
does this mean we should also have `continuation_token` as a reserved parameter name in python mapping

Reply:
Yes, good call, feel free to push that update in this branch